extensions/amp-sidebar/0.1/amp-sidebar.css

```diff
@@ -26,24 +26,33 @@ amp-sidebar {
   overflow-y: auto !important;
   z-index: 9999 !important;
   -webkit-overflow-scrolling: touch;
+  /*
+   * Override layout="nodisplay".
+   * We use translateX and aria-hidden for a11y to hide the sidebar and then
+   * simply translate it back as a separate composition layer.
+   * Note that since sidebar takes owenership of its children, this does
+   * not cause layout for children until sidebar is opened.
+   */
+  display: block !important;
+  will-change: transform;
 }
 
 amp-sidebar[side="left"] {
   left: 0 !important;
-  transform: translateX(-80vw) !important;
+  transform: translateX(-100%) !important;
 }
 
 amp-sidebar[side="right"] {
   right: 0 !important;
-  transform: translateX(80vw) !important;
+  transform: translateX(100%) !important;
 }
 
 amp-sidebar[side][open] {
-  transform: translateX(0vw) !important;
+  transform: translateX(0) !important;
 }
 
-amp-sidebar[side] {
-  transition: transform 0.5s ease;
+amp-sidebar[animate] {
```

Comment:
we consciously decided not to have this property and now this will make it not backward compatible